jwk/handler.go

```diff
@@ -62,6 +67,63 @@ type joseWebKeySetRequest struct {
 	Keys []json.RawMessage `json:"keys"`
 }
 
+// swagger:route GET /.well-known/jwks.json jwk WellKnown
```

Comment:
Awesome, thank you so much! Could you replace the `jwk` tag with `oauth2` and `jwks` and `openid-connect`? :)